.drone.yml

```diff
@@ -8,6 +8,7 @@ build:
     - cask pkg-file ;
       cask install ;
       cask build ;
+      cask exec ecukes --no-win;
```

Comment:
I don't know why `--no-win` isn't working for drone, it worked well for me before on travis.  took the `--no-win` off and it works now.